### ISSUE TITLE
Release a new version?

### ISSUE THREAD

**Issue by tailhook** [zesterer/chumsky#76]
Is there anything that blocks a release for a new version?

I plan to release a library using `chumsky` so would be nice to have current master on the `crates.io`. In particular, I'm using parsers in `delimited_by` (a small problem), and `then_with` (which can't be implemented without private API I think).

**Comment by tailhook** [comment]
Also, would be nice to have #79 in before a release.

**Comment by tailhook** [comment]
Ping @zesterer

**Comment by zesterer** [comment]
I'll try to get time to prepare a new release for tomorrow!

**Comment by zesterer** [comment]
This is currently blocked on #78: I'd like to make sure that this has been solved before proceeding with a release.

**Comment by zesterer** [comment]
Now that #78 has been resolved, I've released `0.8`: https://github.com/zesterer/chumsky/releases/tag/0.8
